packages/tsconfig/react-native-library.json

```diff
@@ -10,6 +10,6 @@
     "module": "ESNext",
     "rootDir": "src",
     "outDir": "dist",
-    "jsx": "react-native"
+    "jsx": "react"
```

Comment:
Dette virket litt merkelig, men det fikk ting til å klikke litt for React Native.